src/components/card/Description.vue

```diff
@@ -332,4 +332,20 @@ h5 {
 #app-sidebar .app-sidebar-header__desc h4 {
 	font-size: 12px !important;
 }
+
+.vue-easymde .cm-s-easymde .cm-link, .cm-image-marker {
+	color: var(--color-text-maxcontrast);
+}
+
+.vue-easymde .cm-s-easymde .cm-image.cm-image-alt-text.cm-link{
+	color: var(--color-main-text);
+}
+
+.vue-easymde .cm-s-easymde .cm-formatting.cm-formatting-image.cm-image.cm-image-alt-text.cm-link{
+	color: var(--color-text-maxcontrast);
+}
+
+.vue-easymde .cm-s-easymde .cm-url {
+	color: #767676 !important;
+}
```

Comment:
The image link color was ok but the basic link color wasn't.

@luka-nextcloud Are you ok with those changes?

```suggestion
.vue-easymde .cm-s-easymde .cm-link {
	color: var(--color-main-text);
}

.vue-easymde .cm-s-easymde .cm-string.cm-url,
.vue-easymde .cm-s-easymde .cm-formatting.cm-link,
.vue-easymde .cm-s-easymde .cm-formatting.cm-url,
.vue-easymde .cm-s-easymde .cm-formatting.cm-image {
	color: var(--color-text-maxcontrast);
}
```

Reply:
Ok, it looks better.